static/skywire-manager-src/src/assets/i18n/en.json

```diff
@@ -384,6 +386,13 @@
         "note": "Note"
       },
 
+      "password-dialog": {
+        "title": "Enter Password",
+        "password": "Password",
+        "info": "You are being asked for a password because a a password was set when the selected entry was created, but the it was not saved for security reasons. You can leave the password empty if needed.",
```

Comment:
It seems there's a redundant `a` in `because a a password was set when the selected entry was created`